envisage/plugins/text_editor/editor/text_editor.py

```diff
@@ -167,15 +168,17 @@ def _obj_changed(self, new):
             with open(new.path, "r", encoding="utf-8") as f:
                 self.text = f.read()
 
-    def _text_changed(self, trait_name, old, new):
+    @observe("text")
+    def _set_dirty(self, event):
```

Comment:
this name might be confused for a property setter for dirty

Reply:
updated the method name to `_update_dirty`